src/auth-service/controllers/token.controller.js

```
@@ -257,16 +257,93 @@
+        ? defaultTenant
+        : req.query.tenant;
+
+      const result = await tokenUtil.verifyToken(request, next);
+      const status = result.status;
+
       if (!res.headersSent) {
-        res.status(status).send(result.message);
+        const acceptsJson =
+          req.headers.accept && req.headers.accept.includes("application/json");
+
+        if (result.success) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)